ethereum/truffle-config.js

```diff
@@ -370,12 +370,13 @@ module.exports = {
 
   compilers: {
     solc: {
-      version: "0.8.4",
+      version: "0.8.19",
```

Comment:
Ah, now I see that you had to modify the truffle config :P
I think if we could somehow tweak truffle to not get in our way, we would be fine. I might take a look at this.

Note that modifying the truffle config implies modifying the wormhole core + token bridge build which we probably won't want to do.

Reply:
Update: if we use [truffle v5.5.14](https://github.com/trufflesuite/truffle/releases/tag/v5.5.14) or higher I think we could avoid compiling generic relayer contracts while building the core with v0.8.4.

Reply:
True, I wonder if this might be okay (to modify the wormhole core + token bridge) because they might be currently built with Truffle anyways? I'm not sure. Probably we should flag this in the description of the big PR (generic-relayer-merge -> main) we end up making.

Reply:
Oh just realized this is the truffle-config. Yeah haha this for sure we'd need to get checked since it modifies the Core/Token bridge